packages/cli/generators/model/index.js

```diff
@@ -154,20 +154,43 @@ module.exports = class ModelGenerator extends ArtifactGenerator {
       },
     ])
       .then(props => {
-        if (typeof props.modelBaseClass === 'object')
+        if (typeof props.modelBaseClass === 'object') {
           props.modelBaseClass = props.modelBaseClass.value;
+        }
 
         Object.assign(this.artifactInfo, props);
         debug(`props after model base class prompt: ${inspect(props)}`);
+        return props;
+      })
+      .catch(err => {
+        debug(`Error during model base class prompt: ${err}`);
+        return this.exit(err);
+      });
+  }
+
+  async promptStrictMode() {
+    if (this.shouldExit()) return false;
+    return this.prompt([
+      {
+        name: 'disabledStrictMode',
+        message: 'Disable strict mode for this model?',
+        type: 'confirm',
+        default: false,
+        when: !this.artifactInfo.disabledStrictMode,
+      },
+    ])
+      .then(setting => {
+        Object.assign(this.artifactInfo, setting);
+
         this.log(
           `Let's add a property to ${chalk.yellow(
```

Comment:
Any reason why we log this message here? I feel it should be in the `promptPropertyName` function. While I also noticed that this code was in the `promptModelBaseClass` function, so I am a little bit confused...

Reply:
It's so that it appears in this order: 
```
lb4 model
? Model class name: model
? Please select the model base class Entity (A persisted model with an ID)
? Disable strict mode for this model? No
Let's add a property to Model
```
instead of:
```
lb4 model
? Model class name: model
? Please select the model base class Entity (A persisted model with an ID)
Let's add a property to Model
? Disable strict mode for this model? No
```